libraries/AP_NavEKF3/AP_NavEKF3_Outputs.cpp

```diff
@@ -607,10 +607,10 @@ void NavEKF3_core::send_status_report(mavlink_channel_t chan) const
     } else {
         temp = 0.0f;
     }
-    const float mag_max = fmaxf(fmaxf(magVar.x,magVar.y),magVar.z);
 
     // send message
-    mavlink_msg_ekf_status_report_send(chan, flags, velVar, posVar, hgtVar, mag_max, temp, tasVar);
```

Comment:
this change is reverting a bug fix that went into master recently

Reply:
fixed

Reply:
thanks Paul